src/script/entity/message/FileAsset.ts

```diff
@@ -35,11 +30,12 @@ type AssetMetaData = (ProtobufAsset.IAudioMetaData | ProtobufAsset.IImageMetaDat
   loudness?: number[];
 };
 
-export class File extends Asset {
-  private readonly original_resource: ko.Observable<AssetRemoteData>;
+export class FileAsset extends Asset {
+  public readonly original_resource: ko.Observable<AssetRemoteData>;
```

Comment:
while we at it.. maybe camelCase?

Reply:
IMHO the PR is already big enough. It would be more difficult to read the PR when also refactoring the naming now.

Reply:
Agree here. Let's focus on functionality rather than style.